src/pybind/mgr/mds_autoscaler/module.py

```diff
@@ -0,0 +1,139 @@
+"""
+Automatically scale MDSs based on status of the file-system using the FSMap
+"""
+
+import logging
+from typing import Optional, List
+from mgr_module import MgrModule
+from ceph.deployment.service_spec import ServiceSpec
+import orchestrator
+import copy
+
+log = logging.getLogger(__name__)
+
+
+class MDSAutoscaler(orchestrator.OrchestratorClientMixin, MgrModule):
+    """
+    MDS autoscaler.
+    """
+    def __init__(self, *args, **kwargs):
+        super(MDSAutoscaler, self).__init__(*args, **kwargs)
```

Comment:
```suggestion
        MgrModule.__init__(*args, **kwargs)
```

With multiple inheritance, `super` is ambiguous. `OrchestratorClientMixin` has no `__init__` method, too.

Reply:
fixed